app/modbus.mjs

```diff
@@ -242,13 +248,26 @@ export const getDeviceInformation = async (modbusClient) => {
     }
 
     result = await mutex.runExclusive(async () => modbusClient.readHoldingRegisters(597, 3))
+    let model = 'unknown'
```

Comment:
Actually, since both `getDeviceFamilyName` and `getDeviceProName` return "unknown" for unknown values, this default here is not needed. Sorry for the confusion. You can leave it like this though, no big deal.